.drone.yml

```diff
@@ -79,26 +79,36 @@ pipeline:
     when:
       branch: master
 
-  trigger_deploy_to_gro_form_dev:
-    image: quay.io/ukhomeofficedigital/drone-trigger:latest
-    drone_server: https://drone.digital.homeoffice.gov.uk
-    repo: UKHomeOffice/kube-gro
-    branch: master
-    deploy_to: gro-form-dev
-    params: APP_IMAGE_TAG=${DRONE_COMMIT_SHA},KUBE_NAMESPACE=gro-form-dev
+  predeploy_clone:
+    image: plugins/git
+    commands:
+      - git clone https://${GITHUB_TOKEN}:x-oauth-basic@github.com/UKHomeOffice/kube-gro.git
     when:
-      branch: master
-      event: push
+      event: deployment
+
+  deploy_to_gro_form_dev:
+    image: quay.io/ukhomeofficedigital/kd:v0.2.2
+    environment:
+      - KUBE_NAMESPACE=gro-form-dev
+    commands:
+      - cd kube-gro
+      - git checkout cf9d58c1740afccad29b3a97b7b7407e66f4862e
```

Comment:
I checked out a specific version of your kubes. There's no tag so I decided to check out a specific sha.

The `kube-gro` folder was created in the previous step where the repo was created.